metron-interface/metron-alerts/src/app/alerts/alerts-list/alerts-list.component.spec.ts

```diff
@@ -31,7 +31,7 @@ import { DialogService } from 'app/service/dialog.service';
 import { Observable } from 'rxjs';
 import { Filter } from 'app/model/filter';
 
-fdescribe('AlertsListComponent', () => {
+describe('AlertsListComponent', () => {
```

Comment:
You should not fix this in the PR. It's fixed in https://github.com/apache/metron/pull/1372 (there's another fdescribe in utils btw)

Reply:
I should remove this to let all tests running in Travis. Don't worry about the conflict, we'll resolve it if it's needed.